modules/RTC/RTCUtils.js

```diff
@@ -876,10 +876,34 @@ class RTCUtils extends Listenable {
                         { googSuspendBelowMinBitrate: true });
                 }
 
-                // There's no reason not to use this for p2p
-                this.p2pPcConstraints.optional.push({
-                    googSuspendBelowMinBitrate: true
-                });
+                /**
+                 * This option is used to enable the suspend video only for
+                 * part of the users on the P2P peer connection. The value of
+                 * the option is the ratio:
+                 * (users with suspended video enabled)/(all users).
+                 *
+                 * Note: The option is not documented because it is temporary
+                 * and only for internal testing purpose.
+                 *
+                 * @type {number}
+                 */
+                const forceP2PSuspendVideoRatio
```

Comment:
i prefer `forceP2PSuspendVideoProbability` to ratio, but maybe just my preference.